exercises/robot-simulator/test/robot_simulator_test.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:robot_simulator/orientation.dart';
+import 'package:robot_simulator/position.dart';
```

Comment:
I believe only a single `.dart` file is uploaded when someone submits a solution (`robot_simulator.dart`). Unless I'm mistaken, `orientation.dart` and `position.dart` should not be required and for the example.dart to pass CI, they can be made exported from `example.dart`

Or, are `orientation.dart` and `position.dart` always provided?

Reply:
`orientation.dart` and `position.dart` are provided in Java and Kotlin tracks for example.

Reply:
@Corentin-Leffy That does not seem to be true. For both the Kotlin and Java tracks, their test files implicitly inherit the source files in the `main` directory. Dart does not do that.

I would prefer we did not force users to solve an exercise in a certain way. The canonical-data.json does not specify these classes to be created, so in **my** opinion it should be left to the students to design the solution files.

Reply:
Do you mean Kotlin/Java tests implicitly import `orientation.dart` and `position.dart` equivalents?. If that's the case they should be dropped in our tests.

This however is also a good opportunity to have `library`, `part`, `part of` directives be used in our example and have them be mentioned as a hint.

If we do not use `Orientation` and `Position` at all, then how do we write type safe tests?

Thoughts?